### ISSUE TITLE
Tabs Design & Feature Request

### ISSUE THREAD

**Issue by mix3d** [vuetifyjs/vuetify#634]
### Design issue
The css design for the tabs do not match the material design docs:
https://material-components-web.appspot.com/tabs.html

Adding the following css styles gets it to the "correct" styles. 
```css
    -webkit-font-smoothing: antialiased;
    font-weight: 500;
    letter-spacing: 0.04em;
    line-height: 1.5rem;
```

Vuetify sets the root html font-size to 14px instead of the default 16px, so most of the MD specs (like `font-size: 0.875rem;`) don't directly apply anymore, but since the tabs don't specify a font-size, its ok in this instance, since the computed 0.875rem is 14.

### Feature Request
Allowing for tabs WITHOUT the included content switcher, allowing for easier inline integration into the toolbar, letting manual page switching work.

**Comment by mix3d** [comment]
I made tabs work in the toolbar, using the following CSS, and ignoring the `v-tabs-content` container. It even works with `grow`, as long as the v-tabs root has a set `width` or `flex` greater than 1.

Now to figure out if the v-tabs-item has the vue-router mixin.

```
  <v-tabs
    id="mobile-tabs-1"
    v-model="active"
    light
  >
    <v-tabs-bar slot="activators">
      <v-tabs-item
        v-for="i in 3"
        :key="i"
        :href="'#mobile-tabs-1-' + i"
        ripple
      >
        Item {{ i }}
      </v-tabs-item>
      <v-tabs-slider></v-tabs-slider>
    </v-tabs-bar>
  </v-tabs>


<style lang="scss">
  .toolbar .tabs{
    width:auto;
    height:100%;
    .tabs__bar{
      height: 100%;
      .tabs__container{
        position: relative;
      }
      .tabs__slider{
        height: 4px;
      }
    }
  }
</style>

**Comment by johnleider** [comment]
This has been fixed as of 0.13 release.
